### DISCUSSION TITLE
Start Orca 2.0 beta

### DISCUSSION THREAD

**Discussion by RaySlovakia** [OrcaSlicer/OrcaSlicer#4604] [General]
Hi.
I have a problem. Every other time that orca starts, it starts without the user settings of the printer. I only see system printers. When I repeatedly restart orca, the user setting is back. When I then quit orca and start it again, the user settings are not there again. And so around. Can you help me? Thx
Sorry for my english

**Comment by Rickthebig** [comment]
I can confirm, happened to me too.
That is why I tried to save the configuration (File->Export->Export config), but that did not work for me either.
Now I hope I do not lose my profiles...

**Reply by RaySlovakia** [reply]
I have solution - start Orca slicer as administrator. This helped to me
